src/Snackbar/index.js

```diff
@@ -0,0 +1,140 @@
+import React, { Component } from 'react'
+import {
+  element,
+  func,
+  number,
+  oneOf,
+  shape,
+  string,
+} from 'prop-types'
+import { isNil } from 'ramda'
+import shortid from 'shortid'
+import classNames from 'classnames'
+import Button from '../Button'
+import {
+  fade,
+  Transition,
+} from '../Transition'
+
+import ThemeConsumer from '../ThemeConsumer'
+
+const consumeTheme = ThemeConsumer('UISnackbar')
+
+/**
+ * The Snackbar component exists to inform the users certain actions he did
+ * in the page.
+ */
+class Snackbar extends Component {
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      visible: true,
+    }
+
+    this.id = shortid.generate()
+
+    if (props.dismissTimeout) {
+      setTimeout(() => {
```

Comment:
Here you could use call `handleDismiss` immediately inside the timeout like this:
```suggestion
      setTimeout(this.handleDismiss, props.dismissTimeout)
```

Reply:
Done!